src/CitMovie.Api/Controller/PromotionalMediaController.cs

```diff
@@ -0,0 +1,122 @@
+using System.Security.Cryptography.X509Certificates;
+
+namespace CitMovie.Api;
+
+[ApiController]
+[Route("api/promotional_media")]
+public class PromotionalMediaController : ControllerBase
+{
+    private readonly IPromotionalMediaManager _manager;
+    private readonly LinkGenerator _linkGenerator;
+
+    public PromotionalMediaController(IPromotionalMediaManager manager, LinkGenerator linkGenerator)
+    {
+        _manager = manager;
+        _linkGenerator = linkGenerator;
+    }
+
+    [HttpGet(Name = nameof(GetPromotionalMedia))]
+    public async Task<IActionResult> GetPromotionalMedia( int page = 0, int pageSize = 10)
```

Comment:
Get promotional media should only return promotional media for a specific release